packages/zoe/zoe.js

```diff
@@ -227,6 +227,7 @@ const makeZoe = async (additionalEndowments = {}) => {
   // securely escrow and get an escrow receipt and payouts in return.
 
   const zoeService = harden({
+    getVersion: () => 'v0.1.0',
```

Comment:
Is there some way to avoid hardcoding this?

Probably not.

Reply:
I looked into some different ways to import package.json, but it seemed to introduce security issues.  There's also `process.env.npm_package_version` but that wasn't working for me.

Reply:
Hm, this seems to be a problem in all languages. In Python I built a thing called [Versioneer](https://github.com/warner/python-versioneer) to grab a version number from the git environment as you do a build.. maybe someone's built something similar for node. Probably as a command that gets run at `npm pack` or `npm build` time, which modifies a file and inserts the current version data just before it gets bundled into the tarball.